.github/workflows/test-performance.yml

```diff
@@ -22,12 +22,13 @@ jobs:
       # we want to run the full build on all os: don't cancel running jobs even if one fails
       fail-fast: false
       matrix:
-        # disable windows-2019 in a first implementation as it always fails. Require more investigations.
+        # disable ${{ vars.RUNNER_WINDOWS }} in a first implementation as it always fails. Require more investigations.
```

Comment:
I suggest we keep _windows-2019_ in comments as the `vars.RUNNER_WINDOWS` value can change at anytime.
This issue we have is for the specific windows version, and the test may work with newer windows version

Reply:
```suggestion
        # disable windows-2019 in a first implementation as it always fails. Require more investigations.
```